<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RP</class>
 <widget class="QWidget" name="RP">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>509</width>
    <height>226</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RP</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(20, 20, 20);</string>
  </property>
  <widget class="QFrame" name="move_frame">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>531</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#202020;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QLabel" name="CS">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>WaitCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size:20px;
color:#fff;</string>
    </property>
    <property name="text">
     <string>Custom Selection</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exit">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>7</y>
      <width>27</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: #202020;
	font-weight:bold;
	font-size:13px;
	border-radius:7px;
	border: 1px solid #787878;
	color:#fff;
}
QPushButton:hover{
	background-color: #141414;
	color:#f75b4d;
}</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QPushButton" name="github">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>10</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: #202020;
	font-size:13px;
	border-radius:5px;
	border: 1px solid #787878;
	color:#fff;
}
QPushButton:hover{
	background-color: #141414;
}</string>
    </property>
    <property name="text">
     <string>github</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="apply_btn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color: #141414;
	font-size:13px;
	border-radius:5px;
	border: 1px solid #787878;
	color:#fff;
}
QPushButton:hover{
	background-color: #212121;
}</string>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="bsic_c">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
	background-color: #141414;
	font-size:13px;
	border-radius:5px;
	border: 1px solid #787878;
	color:#fff;
}
QLineEdit:hover{
	background-color: #212121;
}</string>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="maxLength">
    <number>13</number>
   </property>
   <property name="placeholderText">
    <string>Basic color</string>
   </property>
  </widget>
  <widget class="QPushButton" name="dfl_btn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color: #141414;
	font-size:13px;
	border-radius:5px;
	border: 1px solid #787878;
	color:#fff;
}
QPushButton:hover{
	background-color: #212121;
}</string>
   </property>
   <property name="text">
    <string>Set default settings</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="border_c">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
	background-color: #141414;
	font-size:13px;
	border-radius:5px;
	border: 1px solid #787878;
	color:#fff;
}
QLineEdit:hover{
	background-color: #212121;
}</string>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>13</number>
   </property>
   <property name="placeholderText">
    <string>Border color</string>
   </property>
  </widget>
  <widget class="QFrame" name="refresh_frame">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>60</y>
     <width>171</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 102, 204, 0.2);
border: 1px solid rgb(0, 120, 215);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QLabel" name="CS_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>0</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>WaitCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size:17px;
color:#fff;</string>
    </property>
    <property name="text">
     <string>preview</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="refresh">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>180</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color: #141414;
	font-size:13px;
	border-radius:5px;
	border: 1px solid #787878;
	color:#fff;
}
QPushButton:hover{
	background-color: #212121;
}</string>
   </property>
   <property name="text">
    <string>refresh</string>
   </property>
  </widget>
  <widget class="QFrame" name="notice_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(13, 13, 13);
border-radius:10px;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QLabel" name="CS_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size:10px;
color:#fff;</string>
    </property>
    <property name="text">
     <string>Enter a color in  RGB format example: 0, 102, 204</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="color_picker_button_1">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>90</y>
     <width>21</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255); color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>pictures/color_icon.png</normaloff>pictures/color_icon.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>281</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:11px;
color:#fff;</string>
   </property>
   <property name="text">
    <string>*Notice: when applying a theme you will be thrown out of the windows session</string>
   </property>
   <property name="textFormat">
    <enum>Qt::TextFormat::PlainText</enum>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
  </widget>
  <widget class="QPushButton" name="color_picker_button_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>120</y>
     <width>21</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255); color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>pictures/color_icon.png</normaloff>pictures/color_icon.png</iconset>
   </property>
  </widget>
  <zorder>notice_2</zorder>
  <zorder>move_frame</zorder>
  <zorder>apply_btn</zorder>
  <zorder>bsic_c</zorder>
  <zorder>dfl_btn</zorder>
  <zorder>border_c</zorder>
  <zorder>refresh_frame</zorder>
  <zorder>refresh</zorder>
  <zorder>color_picker_button_1</zorder>
  <zorder>label</zorder>
  <zorder>color_picker_button_2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
